### PR TITLE
Implement UI for remaining smart plug details.

### PR DESCRIPTION
**Pull request by mrstegeman**
In addition, add some constants that the Z-Wave and Zigbee adapters
were looking for.

### CODE REVIEW

**Comment by benfrancis** [comment]
I actually intentionally left out the other properties from the UI for simplicity, particularly the level property which Dave noted is advertised by some devices even though it doesn't actually work.

File: static/js/level-detail.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ LevelDetail.prototype.attach = function() {
 }
 
 LevelDetail.prototype.view = function() {
-  let level = this.thing.properties.level;
+  const level = this.thing.properties.level;
```
Comment (benfrancis) [inline static/js/level-detail.js]: How is this a constant?

Comment (mrstegeman) [reply static/js/level-detail.js]: `level` is never redefined.

**Comment by mrstegeman** [comment]
@benfrancis The properties are all conditionally shown. If they're not in the property description, they won't be shown. I can confirm that my Z-Wave plug did not dim properly. I think the proper solution would be to not report `level` if it's not functional.

**Comment by benfrancis** [comment]
@mrstegeman OK, I think Dave was saying that the problem is the Z-Wave device reports that it has a multilevel switch but it actually just acts as a binary switch (or whatever the Z-Wave terms are), but there's no way for the adapter to know that it doesn't work.

**Comment by mrstegeman** [comment]
@benfrancis Maybe we just disable the `level` property on Z-Wave devices until we get that figured out?

**Comment by dhylands** [comment]
The ZW096 module is a relay device and as such can't dim. The ZW099 module is an actual dimmer and does dim.

**Comment by dhylands** [comment]
This is a case where the adapter really needs to say, this device doesn't behave this way, so I'll adjust the properties. The ZW096 is a smartPlug with no level (the device says it supports the level command and it it does in fact respond to the level command, but they don't cause any changes to the physical device). The ZW096 is a smartPlug where the level should work properly.

What we really need is something the linux kernel calls 'quirks' (or special cases) and the adapter would just not add the level property for a ZW096

**Comment by mrstegeman** [comment]
Z-Wave issues addressed here: https://github.com/mozilla-iot/zwave-adapter/pull/1

**Pull request merged**